docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: "1"
```

Comment:
The Docker Compose version is set to `1`, which is outdated. Consider updating it to a more recent version, such as `3.8`, to take advantage of newer features and improvements.